search-spec/openapi.yml

```diff
@@ -0,0 +1,229 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: UP42 Tasking
+  license:
+    name: MIT
+servers:
+  - url: /v1
+paths:
+  /search:
+    post:
+      summary: Get possible tasking options for spatial temporal query
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/SearchQuery"
```

Comment:
This looks an awful lot like a STAC search request. 

Which makes sense, because tasking slots are spatio-temporal in nature.

Why are we introducing a separate spec instead of a STAC extension?